mods/species/vox/organs_vox.dm

```diff
@@ -200,25 +200,29 @@
 /obj/item/organ/internal/voxstack/proc/backup_inviable()
 	return 	(!istype(backup) || backup == owner.mind || (backup.current && backup.current.stat != DEAD))
 
-/obj/item/organ/internal/voxstack/replaced()
+/obj/item/organ/internal/voxstack/on_replacement()
 	if(!..()) return 0
 	if(prompting) // Don't spam the player with twenty dialogs because someone doesn't know what they're doing or panicking.
 		return 0
+	
+	//Need spawn here so that this interactive bit doesn't lock up init
 	if(owner && !backup_inviable())
-		var/current_owner = owner
+		addtimer(CALLBACK(src, .proc/prompt_revive_callback, owner), 0)
```

Comment:
If the goal is just to unblock (i.e. returning the 1 on the next line before calling this proc is not important), invoking directly and setting `waitfor FALSE` on the callback would be preferred. This kind of timer/spawn pattern is often used to hack ordering--and that's usually highly undesirable.

Reply:
The reason why I'm invoking it like this, is because there's a sleep in "prompt_revive_callback". This code gets called in "life()" sometimes, and dreamchecker doesn't like it, rightfully so. Would calling "prompt_revive_callback()" with its "waitfor FALSE" set work in that case?

Reply:
Yes, setting `waitfor = FALSE` on the callback and invoking it directly here should be fine then, and the linter should pick that up.

Reply:
Gotcha